test/FunctionalTests/Microsoft.OData.Core.Tests/IntegrationTests/Evaluation/AutoComputePayloadMetadataInJsonIntegrationTests.cs

```diff
@@ -2149,7 +2149,7 @@ public void ReadingInMinialMetadataModeWithContainedElementShouldBeAbleToGenerat
                 "} ] }";
 
             InMemoryMessage message = new InMemoryMessage();
-            message.SetHeader("Content-Type", "application/json;odata.metadata=mini");
+            message.SetHeader("Content-Type", "application/json;odata.metadata=minimal");

```

Comment:
This test doesn't not seem to be related to change that you made.

Reply:
I just found this while try to fix my related Issue. In my opinion this has been a typo